source/sqlsrv/conn.cpp

```diff
@@ -845,13 +844,15 @@ PHP_FUNCTION( sqlsrv_close )
 
 void __cdecl sqlsrv_conn_dtor( _Inout_ zend_resource *rsrc TSRMLS_DC )
 {
-    LOG_FUNCTION( "sqlsrv_conn_dtor" );
+    // Without sqlsrv_close(), this function is invoked by php during the final clean up stage. 
+    // To prevent memory/resource leaks, no more logging at this point.
+    //LOG_FUNCTION( "sqlsrv_conn_dtor" );
 
     // get the structure
     ss_sqlsrv_conn *conn = static_cast<ss_sqlsrv_conn*>( rsrc->ptr );
     SQLSRV_ASSERT( conn != NULL, "sqlsrv_conn_dtor: connection was null");
 
-    SET_FUNCTION_NAME( *conn );
+    conn->set_func("sqlsrv_conn_dtor");
```

Comment:
Would \_FN\_ work here as well?

Reply:
Not this one because the macro LOG_FUNCTION is commented but I can certainly use `__func__`
Thanks :)